docs/tutorials/demo_data_science_project.ipynb

```diff
@@ -9,42 +9,67 @@
    "source": [
     "# Introduction\n",
     "\n",
-    "In this notebook, we will complete a small end-to-end data science tutorial that employs `lakeFS-spec` for data versioning. We will use weather data to train a random forest classifier to predict whether a given day from now is a raining day given the current weather.\n",
+    "In this notebook, we will complete a small end-to-end data science tutorial that employs lakeFS-spec for data versioning. We will use versioned weather data to train a decision tree classifier to predict whether it is raining tomorrow given the current weather.\n",
     "\n",
     "We will do the following:\n",
     "\n",
     "* Environment setup\n",
-    "* lakeFS setup\n",
-    "* Data ingestion\n",
-    "    * Transactions\n",
-    "    * PUT a file\n",
+    "* LakeFS setup\n",
+    "* Authenticate with the lakeFS client\n",
```

Comment:
```suggestion
    "* Authenticating with the lakeFS server\n",
```

since authentication is done *with* the authority that's verifying you, which is the lakeFS server in this case.